lib/api/objectPutPart.js

```diff
@@ -63,7 +74,7 @@ export default function objectPutPart(authInfo, request, log, cb) {
     if (partNumber > 10000) {
         return cb(errors.TooManyParts);
     }
-    if (!Number.isInteger(partNumber)) {
+    if (!Number.isInteger(partNumber) || partNumber < 1) {
```

Comment:
I was viewing it as a feature actually (it's in its own commit)  :)
I'll drop it here and do a separate pr against 6.2.